recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ test: {}
 about:
   home: https://github.com/conda-forge/blas-feedstock
   license: BSD 3-clause
+  license_file: {{ environ['RECIPE_DIR'] }}/LICENSE.txt
```

Comment:
I guess the linter is balking at this. We have used `environ` successfully elsewhere. However, it has normally been in a list like test dependencies. We should raise this to the linter issue tracker. In the interim, I'm going to try and come up with something more friendly for the linter.